src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Symfony\Component\Messenger\Stamp\RedeliveryStamp" exclusion-policy="ALL">
+        <property name="message" type="string" expose="true" />
+        <property name="code" type="int" expose="true" />
+        <property name="previous" type="Throwable" expose="true" />
```

Comment:
don't think this type exists, maybe just exclude it?

Reply:
It exists, just as an interface since PHP 7. Yeah, I will exclude it.

Reply:
I meant in JMS, if there is no mapping file, I think it fails to serialize